docs/getting-started/application/patterns.rst

```diff
@@ -282,303 +176,26 @@ We can combine field access control with our record level access control
     We use the ``oso.query`` method in this example to query a rule other than
     ``allow``.
 
-.. todo::
-
-    relevant link & this is incorrect with our API now!
-
-In this example, we evaluated both record & column level access control after
-data fetch.  However, it may be more efficient to use column level access
-control to only load the columns the user can access:
-
-.. code-block:: python
-    :caption: :fab:`python`
-
-    from oso import Variable
-
-    def get_expense(user, expense_id):
-        # Query oso for all fields allowed for this user.
-        allowed_fields = oso.query_rule("allow_field",
-                                   user, "view", expense, Variable("field"))
-        # Convert the returned query response into a list of fields
-        allowed_fields = [r["field"] for r in allowed_fields]
-        allowed_fields_sql = db.sql_escape(allowed_fields.join(", "))
-
-        expense = db.fetch(
-            f"SELECT {allowed_fields_sql} FROM expenses WHERE id = %",
-            expense_id)
```

Comment:
This is actually very cool but definitely out of place without more explanation of stuff like `Variable`.